xml/System.Security.Cryptography.Xml/KeyInfoRetrievalMethod.xml

```diff
@@ -140,7 +140,7 @@
       <Docs>
         <param name="strUri">The Uniform Resource Identifier (URI) of the information to be referenced by the new instance of <see cref="T:System.Security.Cryptography.Xml.KeyInfoRetrievalMethod" />.</param>
         <param name="typeName">The URI that describes the type of data to retrieve.</param>
-        <param name="strType">To be added.</param>
+        <param name="strType">The URI that describes the type of data to retrieve.</param>
```

Comment:
@bartonjs @GrabYourPitchforks this was detected by DocsPortingTool as an argument that got renamed, and had to manually choose which was the old argument name. I am not entirely sure if `strType` was `typeName` or `strUri`, but it made more sense to me to choose the description from `typeName`. Was this the right choice?

Reply:
It's called `typeName` in both .NET Framework and .NET (nee Core), for some reason it was `strType` in Xamarin.